2025-03-10 16:22:21,682 INFO    MainThread:2788 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-03-10 16:22:21,682 INFO    MainThread:2788 [wandb_setup.py:_flush():67] Configure stats pid to 2788
2025-03-10 16:22:21,682 INFO    MainThread:2788 [wandb_setup.py:_flush():67] Loading settings from C:\Users\Ashutosh Patidar\.config\wandb\settings
2025-03-10 16:22:21,682 INFO    MainThread:2788 [wandb_setup.py:_flush():67] Loading settings from C:\Users\Ashutosh Patidar\OneDrive\Documents\GitHub\Assignment_1_Deep_Learning\wandb\settings
2025-03-10 16:22:21,682 INFO    MainThread:2788 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-10 16:22:21,682 INFO    MainThread:2788 [wandb_init.py:setup_run_log_directory():647] Logging user logs to C:\Users\Ashutosh Patidar\OneDrive\Documents\GitHub\Assignment_1_Deep_Learning\wandb\run-20250310_162221-l3ke0xas\logs\debug.log
2025-03-10 16:22:21,682 INFO    MainThread:2788 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to C:\Users\Ashutosh Patidar\OneDrive\Documents\GitHub\Assignment_1_Deep_Learning\wandb\run-20250310_162221-l3ke0xas\logs\debug-internal.log
2025-03-10 16:22:21,698 INFO    MainThread:2788 [wandb_init.py:init():761] calling init triggers
2025-03-10 16:22:21,698 INFO    MainThread:2788 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'dataset': 'mnist', 'hidden_sizes': [256, 128], 'activation_function': 'relu', 'learning_rate': 0.005, 'epochs': 20, 'batch_size': 64, 'optimizer': 'adam', 'beta_1': 0.9, 'beta_2': 0.999, 'loss_function': 'categorical_cross_entropy', 'weight_decay': 0.0, 'epsilon': 1e-08, '_wandb': {}}
2025-03-10 16:22:21,698 INFO    MainThread:2788 [wandb_init.py:init():784] starting backend
2025-03-10 16:22:21,698 INFO    MainThread:2788 [wandb_init.py:init():788] sending inform_init request
2025-03-10 16:22:21,790 INFO    MainThread:2788 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2025-03-10 16:22:21,791 INFO    MainThread:2788 [wandb_init.py:init():803] backend started and connected
2025-03-10 16:22:21,794 INFO    MainThread:2788 [wandb_init.py:init():896] updated telemetry
2025-03-10 16:22:21,894 INFO    MainThread:2788 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-03-10 16:22:22,409 INFO    MainThread:2788 [wandb_init.py:init():995] starting run threads in backend
2025-03-10 16:22:23,892 INFO    MainThread:2788 [wandb_run.py:_console_start():2377] atexit reg
2025-03-10 16:22:23,893 INFO    MainThread:2788 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-10 16:22:23,893 INFO    MainThread:2788 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-10 16:22:23,893 INFO    MainThread:2788 [wandb_run.py:_redirect():2317] Redirects installed.
2025-03-10 16:22:23,901 INFO    MainThread:2788 [wandb_init.py:init():1037] run started, returning control to user process
2025-03-10 16:31:34,178 WARNING MsgRouterThr:2788 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "C:\Users\Ashutosh Patidar\AppData\Roaming\Python\Python312\site-packages\wandb\sdk\interface\router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ashutosh Patidar\AppData\Roaming\Python\Python312\site-packages\wandb\sdk\lib\sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ashutosh Patidar\AppData\Roaming\Python\Python312\site-packages\wandb\sdk\lib\sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Ashutosh Patidar\AppData\Roaming\Python\Python312\site-packages\wandb\sdk\interface\router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ashutosh Patidar\AppData\Roaming\Python\Python312\site-packages\wandb\sdk\interface\router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-03-10 16:31:34,300 INFO    MsgRouterThr:2788 [mailbox.py:close():115] Closing mailbox, abandoning 1 handles.
